include/inviwo/core/properties/ordinalproperty.h

```diff
@@ -224,6 +224,54 @@ IVW_CORE_API OrdinalPropertyState<vec3> ordinalLight(
     const vec3& pos, float min = -100.0, float max = 100.0,
     InvalidationLevel invalidationLevel = InvalidationLevel::InvalidOutput);
 
+template <typename T, typename U>
+OrdinalPropertyState<T> ordinalSymmetricVector(
+    const T& val, const U& minMax,
+    InvalidationLevel invalidationLevel = InvalidationLevel::InvalidOutput,
+    PropertySemantics semantics = PropertySemantics::SpinBox) {
+
+    return {val,
+            decltype(val){-minMax},
+            ConstraintBehavior::Ignore,
+            decltype(val){minMax},
+            ConstraintBehavior::Ignore,
+            decltype(val){0.1},
+            invalidationLevel,
+            semantics};
+}
+
+template <typename T = size_t, typename U = T>
+OrdinalPropertyState<T> ordinalCount(
```

Comment:
this way it it is possible to initiate the bound with an int even if T is a size_t if both was T you have to write `OrdinalCount(size3_t{5,5,5}, size_t{100})` instead of just `OrdinalCount(size3_t{5,5,5}, 100)` which I found more intuitive. I can change it if you like, but when using it I would bet that everyone will use the later version and get an error which is a bit cryptic, and then have to change to the first version, this way the second version just works.

Reply:
I see, I would prefer avoiding cryptic error messages :)

Reply:
Perhaps you could add the explanation above as a comment